packages/office-ui-fabric-react/src/components/Link/Link.base.tsx

```diff
@@ -67,7 +67,7 @@ export class LinkBase extends BaseComponent<ILinkProps, any> implements ILink {
     // Deconstruct the props so we remove props like `as`, `theme` and `styles`
     // as those will always be removed. We also take some props that are optional
     // based on the RootType.
-    const { children, as, disabled, target, href, theme, getStyles, styles, ...restProps } = props;
+    const { children, as, disabled, target, href, theme, getStyles, styles, componentRef, ...restProps } = props;
```

Comment:
hrmmm. Shouldn't this just be `getNativeProps`? it seems like as soon as `foo` prop is added to `Link` the issue will resurface.

Reply:
Good point, except that wouldn't work if you pass a component as the `as`. But we could probably check for that. 

I think the best would be if we do this fancy typing thing we once had on email. But that would only work for TS folks. 

I'll have another look at this and use `getNativeProps` for when you pass a string.